src/components/text/headlineConsts.js

```diff
@@ -0,0 +1,27 @@
+export const HEADLINE_TYPE = {
```

Comment:
I think we should keep it in one file to make import easier - or move those 2 files to separate directory `headlines` and add index.js that reexports from both of them.

Reply:
@aju yeah, sounds like a good idea. But maybe I will keep it for later? maybe we could use one config for headlines and body text somehow. 
I made a refactor of logic for headlines. If you have a moment, please have a look and tell me what do you think. 🐯